### PR TITLE
Add an demo endpoint for Eyevine

### PR DESCRIPTION
**Pull request by marialivia16**
Adding a `/images/example` endpoint for demo purposes that always return the same image.

### CODE REVIEW

**Comment by marialivia16** [comment]
[Create a demo endpoint for Eyevine to hit](https://trello.com/c/NeCD0g54/42-create-a-demo-endpoint-for-eyevine-to-hit)

File: media-api/conf/example-image.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
+{
```
Comment (akash1810) [inline media-api/conf/example-image.json]: any reason why we're not fetching this from elasticsearch?

Comment (marialivia16) [reply media-api/conf/example-image.json]: The idea was to have the same image in all environments without having to worry whether it exists in ES or whether the ids are different for example.

**Comment by LATaylor-guardian** [comment]
👍 If this is representative of what we expect Eyevine to receive then let's get this merged and over to them so they can begin the work on their side 🚢

File: media-api/conf/example-image.json (hunk 1 above)
Comment (akash1810) [reply media-api/conf/example-image.json]: makes sense 👍 however, eyevine would need to be able to download the image to fully test the process their end. Could we return data for a real image, with links to our cdn etc?

Comment (marialivia16) [reply media-api/conf/example-image.json]: Will do, makes sense.

**Pull request merged**